::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 27 Jan 2018 19:55:31 +0100
Subject: [PATCH] Update IA for vPDF support

- move gtw tenant addrs from {6,7,9} to {124,125,126}
  (e.g. 10.0.1.6 -> 10.0.1.124 for ericsson-virtual* PODs);
- add gtw external (public) addresses, e.g. 10.16.0.124;
- add ctl external (public) addresses, e.g. 10.16.0.11;

JIRA: FUEL-322

Change-Id: Ice73aa3985831dca7427e496850d0865db71477e
Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 config/installers/fuel/pod_config.yml.j2 | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/config/installers/fuel/pod_config.yml.j2 b/config/installers/fuel/pod_config.yml.j2
index b3fd9c4..65336a5 100644
--- a/config/installers/fuel/pod_config.yml.j2
+++ b/config/installers/fuel/pod_config.yml.j2
@@ -59,9 +59,12 @@ parameters:
     opnfv_openstack_gateway_node01_address: {{ net_mgmt | ipaddr_index(124) }}
     opnfv_openstack_gateway_node02_address: {{ net_mgmt | ipaddr_index(125) }}
     opnfv_openstack_gateway_node03_address: {{ net_mgmt | ipaddr_index(126) }}
-    opnfv_openstack_gateway_node01_tenant_address: {{ net_private | ipaddr_index(6) }}
-    opnfv_openstack_gateway_node02_tenant_address: {{ net_private | ipaddr_index(7) }}
-    opnfv_openstack_gateway_node03_tenant_address: {{ net_private | ipaddr_index(9) }}
+    opnfv_openstack_gateway_node01_tenant_address: {{ net_private | ipaddr_index(124) }}
+    opnfv_openstack_gateway_node02_tenant_address: {{ net_private | ipaddr_index(125) }}
+    opnfv_openstack_gateway_node03_tenant_address: {{ net_private | ipaddr_index(126) }}
+    opnfv_openstack_gateway_node01_external_address: {{ net_public | ipaddr_index(124) }}
+    opnfv_openstack_gateway_node02_external_address: {{ net_public | ipaddr_index(125) }}
+    opnfv_openstack_gateway_node03_external_address: {{ net_public | ipaddr_index(126) }}
     opnfv_openstack_proxy_address: {{ net_public | ipaddr_index(103) }}
     opnfv_openstack_proxy_node01_address: {{ net_public | ipaddr_index(104) }}
     opnfv_openstack_proxy_node02_address: {{ net_public | ipaddr_index(105) }}
@@ -72,6 +75,9 @@ parameters:
     opnfv_openstack_control_node01_address: {{ net_mgmt | ipaddr_index(11) }}
     opnfv_openstack_control_node02_address: {{ net_mgmt | ipaddr_index(12) }}
     opnfv_openstack_control_node03_address: {{ net_mgmt | ipaddr_index(13) }}
+    opnfv_openstack_control_node01_external_address: {{ net_public | ipaddr_index(11) }}
+    opnfv_openstack_control_node02_external_address: {{ net_public | ipaddr_index(12) }}
+    opnfv_openstack_control_node03_external_address: {{ net_public | ipaddr_index(13) }}
     opnfv_openstack_database_address: {{ net_mgmt | ipaddr_index(50) }}
     opnfv_openstack_database_node01_address: {{ net_mgmt | ipaddr_index(51) }}
     opnfv_openstack_database_node02_address: {{ net_mgmt | ipaddr_index(52) }}
